packages/rich-text-editor/src/RichTextEditor/schema.ts

```diff
@@ -0,0 +1,51 @@
+import { MarkSpec, NodeSpec, Schema } from "prosemirror-model"
+import boldIcon from "@kaizen/component-library/icons/bold.icon.svg"
+import italicIcon from "@kaizen/component-library/icons/italics.icon.svg"
+import underlineIcon from "@kaizen/component-library/icons/underline.icon.svg"
+import {
+  nodes as coreNodes,
+  marks as coreMarks,
+} from "@cultureamp/rich-text-toolkit"
+
+export const nodes: NodeSpec = {
+  doc: coreNodes.doc,
+  paragraph: coreNodes.paragraph,
+  text: coreNodes.text,
+  // eslint-disable-next-line camelcase
+  hard_break: coreNodes.hard_break,
+}
+
+export const marks: MarkSpec = {
```

Comment:
I've left some marks stuff in here from my copy+paste, even though we're not really using it yet, because `RichTextEditor` is using the `createSchemaFromControls` function below to create the schema.